css/index.css

```diff
@@ -1,3 +1,410 @@
+                                                    justify-content: center;
+                                                    width: 800px;
+                                                    margin-left: inherit;
+                                                }
+                                                
+                                                .middles {
+                                                    width: 880px;
+                                                }
+                                                
+                                                span {
+                                                    text-align: center;
+                                                    margin-left: 85px;
+                                                }
+                                                
+                                                anchor {
+                                                    order: 0;
+                                                }
+                                                
+                                                .a-logo {
+                                                    order: 0;
+                                                    width: 25%;
+                                                    text-align: center
+                                                }
+                                                /* Media Query Tablet */
+                                                
+                                                @media only screen and (max-width: 800px) {
+                                                    body {
+                                                        width: 100%;
+                                                        margin: 0 auto;
+                                                        margin-right: 550px;
+                                                    }
+                                                    .container {
+                                                        width: 100%;
+                                                    }
+                                                    nav {
+                                                        flex-wrap: wrap;
+                                                        justify-content: space-around;
+                                                        margin-left: 10px;
+                                                    }
+                                                    nav a {
+                                                        padding: 15px;
+                                                    }
+                                                    nav a:last child {
+                                                        order: -1;
+                                                        width: 100%;
+                                                    }
+                                                    .imm {
```

Comment:
Nice job hiding the image on small screens. Don't forget that you can use flexbox to center that button as well.